controller/namedspaces.go

```diff
@@ -52,7 +53,35 @@ func (c *NamedspacesController) Show(ctx *app.ShowNamedspacesContext) error {
 	return ctx.OK(res)
 }
 
-func loadKeyCloakIdentityByUserName(ctx *app.ShowNamedspacesContext, appl application.Application, username string) (*account.Identity, error) {
+func (c *NamedspacesController) List(ctx *app.ListNamedspacesContext) error {
+	offset, limit := computePagingLimts(ctx.PageOffset, ctx.PageLimit)
+	if ctx.UserName == "" {
+		return jsonapi.JSONErrorResponse(ctx, goa.ErrNotFound("not found, userName=%v", ctx.UserName))
+	}
+
+	return application.Transactional(c.db, func(appl application.Application) error {
+		identity, err := loadKeyCloakIdentityByUserName(ctx, appl, ctx.UserName)
+		if err != nil {
+			return jsonapi.JSONErrorResponse(ctx, goa.ErrNotFound("not found, userName=%v", ctx.UserName))
```

Comment:
Looks good but let's add the parent error message since it can give more details on what is wrong: fmt.Sprintf("not found, userName=%v %v", ctx.UserName, err.Error())